wrapper.py

```diff
@@ -89,31 +94,31 @@ def min_alpha_beta(self, deep, alpha, beta):
 
         return minv
 
+    @timed
     def play_alpha_beta(self):
-        while True:
-            if self.is_end():
-                if self.current_state[6] - self.current_state[13] > 0:
-                    return self.player
-                else:
-                    return 3 - self.player
-
-            (m, h) = self.max_alpha_beta(0, -2000, 2000)
-            val = self.current_state[h]
-            self.current_state[h] = 0
-            pos = h + 1
-            for i in range(0, val):
-                if pos == 13:
-                    pos = 0
-                self.current_state[pos] += 1
-                pos += 1
+        while not self.is_end():
+            _, h = self.max_alpha_beta(0, -2000, 2000)
+            self._transfer_stones(h)
+
+            self.current_state[:self.__h], self.current_state[self.__h:] = \
+                self.current_state[self.__h:], self.current_state[:self.__h]
+            self.player ^= 1
+        # print(self.current_state[6] == self.current_state[13])
+        # changing sign affects the results table
+        # bcoz the equality isn't taken into account
+        return self.player ^ (self.current_state[6] > self.current_state[13])
```

Comment:
equality is completely ignored (as in original)
changing the comparison from ">" to "<" (and changing player accordingly) can drastically change the score board